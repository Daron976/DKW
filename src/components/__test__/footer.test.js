import { render, screen } from '@testing-library/react';
import renderer from 'react-test-renderer';
import Footer from '../footer';

describe('Footer component', () => {
  test('header renders as exepcted', async () => {
    const snapfooter = renderer.create(<Footer />)
      .toJSON();
    expect(snapfooter).toMatchSnapshot();
  });
  test('Footer is present in the Dom', async () => {
    render(<Footer />);

    const calcComponent = screen.getByTestId('footer');

    expect(calcComponent).toBeInTheDocument; //eslint-disable-line
  });
});
