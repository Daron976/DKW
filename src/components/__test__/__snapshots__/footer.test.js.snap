// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component header renders as exepcted 1`] = `
<footer
  className="footer-wrapper"
  data-testid="footer"
>
  <div
    className="footer-content"
  >
    <p
      className="copyright-text"
    >
      Created with mathematical inspirations
    </p>
  </div>
</footer>
`;
